src/hotspot/cpu/x86/vm_version_x86.hpp

```diff
@@ -653,7 +653,7 @@ class VM_Version : public Abstract_VM_Version {
   //       3   -  386
   //       4   -  486
   //       5   -  Pentium
-  //       6   -  PentiumPro, Pentium II, Celeron, Xeon, Pentium III, Athlon,
+  //       6   -  PentiumPro, Pentium II, Celer;on, Xeon, Pentium III, Athlon,
```

Comment:
I haven't heard of that processor family. Nor does intel talk about it: https://www.intel.com/content/www/us/en/products/details/processors/celeron.html

Reply:
Oh, maybe because it's not released yet. 

sorry, that was unintentional-typo; Fixed :)